jicamargo/helloworld-frontend
# 🚀 Front-End with React/Redux for Hello World Project**

🚀 Front-End with React/Redux for Hello World Project**

**Description**

This PR introduces the front-end component of the Hello World project. 🌍

## 🤝 **Link to "Partner" Back-End Project**

The front-end interacts seamlessly with the Hello World Back-End project, which is a Ruby on Rails API backed by a PostgreSQL database. 
***[Visit Back-end repository here](https://github.com/jicamargo/helloworld_backend)***

**Changes Made**

- Created React-Redux application for the front-end.
- Implemented Redux actions and reducers to fetch and manage greeting messages.
- Developed React components to display greetings to the user.
- Integrated React Router for navigation within the application.
- Ensured compatibility with the Hello World Back-End for greeting retrieval.

📝 **How to Test**

1. Clone this repository.
2. Install Node.js and npm if not already installed.
3. Run `npm install` to install project dependencies.
4. Start the development server with `npm start`.

Ensure the Hello World Back-End is running locally on the specified port (default: 3000) to handle API requests.

Thank you for your suggestions and Review! 👍🏽
